extensions/assistedinject/pom.xml

```diff
@@ -13,4 +13,21 @@
 
   <name>Google Guice - Extensions - AssistedInject</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.google.inject</groupId>
+      <artifactId>guice</artifactId>
```

Comment:
All extensions depend on Guice core, so it looks odd to have to repeat this in each pom rather than simply declare it once in the extensions parent pom as it was before.